Sources/Common/Background Queue/QueueStorage.swift

```diff
@@ -24,99 +24,76 @@ import Foundation
  if a customer app creates lots of tasks for an app that is in Airplane mode, for example.
  */
 public protocol QueueStorage: AutoMockable {
-    func getInventory() -> [QueueTaskMetadata]
-    func get(storageId: String) -> QueueTask?
-    func delete(storageId: String) -> Bool
-    func deleteExpired() -> [QueueTaskMetadata]
+    func getInventory(siteId: String) -> [QueueTaskMetadata]
+    func get(storageId: String, siteId: String) -> QueueTask?
+    func delete(storageId: String, siteId: String) -> Bool
 }
 
 // sourcery: InjectRegister = "QueueStorage"
 public class FileManagerQueueStorage: QueueStorage {
     private let fileStorage: FileStorage
     private let jsonAdapter: JsonAdapter
-    let siteId: String
-    private let sdkConfig: SdkConfig
     let logger: Logger
     let dateUtil: DateUtil
     private var inventoryStore: QueueInventoryMemoryStore
 
     let lock: Lock
 
-    private var inventory: [QueueTaskMetadata]? {
```

Comment:
All of the `inventory` get/set code moved into `getInventory` and `saveInventory` functions. Since we now need a `siteId` to get and save inventory.